template/src/ontology/Makefile.jinja2

```diff
@@ -977,6 +977,17 @@ GITHUB_RELEASE_PYTHON=make-release-assets.py
 public_release:
 	ls -alt $(ASSETS)
 	$(GITHUB_RELEASE_PYTHON) --release $(TAGNAME) $(RELEASEFILES)
+{% elif project.public_release == 'gh_main'  %}
+
+RELEASE_ASSETS_RELEASE_DIR=$(foreach n,$(RELEASE_ASSETS), ../../$(n))
```

Comment:
Minor nitpicking, but I dislike the fact this variable is named `*_DIR` while it is intended to list release _files_. It makes the intention not clear at first.

Reply:
Thanks, renamed it! The name meant: release assets after they are being copied to the release dir. :P